scripts/build-controller.sh

```diff
@@ -98,6 +96,18 @@ if [[ ! -d $SERVICE_CONTROLLER_SOURCE_PATH ]]; then
     exit 1
 fi
 
+BOILERPLATE_TXT_PATH="$ROOT_DIR/templates/boilerplate.txt"
+DEFAULT_TEMPLATE_DIRS="$ROOT_DIR/templates"
+# If the service controller source repository has a templates/ directory, add
```

Comment:
very minor: Maybe adding a comment regarding the priority of template override here will be helpful. If script run does not produce desired results, the developer first comes to check this script rather than reading the priority related comment in go files.

Reply:
@vijtrip2 great suggestion. added!